tests/test_production_cost_speed_optimizer_safe.py

```
@@ -0,0 +1,47 @@
+
+def test_safe_installer_enables_only_external_only_repair_skip(monkeypatch):
+    from korgan import production_cost_speed_optimizer_safe as safe
+    from korgan.legal import corpus_refresh
+
+    calls: list[str] = []
+
+    monkeypatch.setattr(safe, "_INSTALLED", False)
+    monkeypatch.setattr(
+        safe.optimizer,
+        "_progressive_refresh_factory",
+        lambda current, loader: current,
```

Ruff (0.16.2)

[warning] Unused lambda argument: `loader`

(ARG005)